src/components/Spinner/readme.md

```diff
@@ -58,15 +54,28 @@ const spinner = (
         </h1>
```

Comment:
Please add theme support for this text
<img width="1254" alt="Screen Shot 2019-12-19 at 11 58 52 PM" src="https://user-images.githubusercontent.com/32083556/71233841-66ac8600-22bc-11ea-9b5b-81f744817502.png">